app/app.model.js

```diff
@@ -69,6 +69,34 @@ const checkIfExists = (table, column, value) => {
     })
 }
 
+const insertComment = (username, body, article_id) => {
+    return checkIfExists("articles", "article_id", article_id)
+    .then(() => {
+        return db.query(`
+            INSERT INTO comments (
+                author,
+                body,
+                article_id,
+                votes,
+                created_at
+    
+            )
+            VALUES (
+                $1,
+                $2,
+                $3,
+                DEFAULT,
+                DEFAULT
+            )
+            RETURNING *
+        `,
+        [username, body, article_id])
+    })
```

Comment:
For votes and created_at, you dont need to specify these to be DEFAULT, in this case simply not having them in the INSERT INTO statement will automatically use the default values.